qiskit/transpiler/preset_passmanagers/generate_preset_pass_manager.py

```diff
@@ -396,7 +396,7 @@ def _parse_coupling_map(coupling_map, backend):
 
 
 def _parse_instruction_durations(backend, dt):
-    """Create a list of ``InstructionDuration``s populated from the backend."""
+    """Create a list of ``InstructionDurations`` objects populated from the backend."""
```

Comment:
There's an accidental `s` introduced here
```suggestion
    """Create a list of ``InstructionDuration`` objects populated from the backend."""
```